sdk/communication/communication-call-automation/src/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "tabWidth": 2,
```

Comment:
can you verify if this is part of your environment code?

Reply:
removed